src/app/static/src/tests/modelOptions/mutations.test.ts

```diff
@@ -101,7 +101,7 @@ describe("Model run options mutations", () => {
                         {
                             label: "g1",
                             controls: [
-                                {...mockControl, value: 10},
+                                {...mockControl, name: "n1", value: 20},
```

Comment:
This test was sharing control names between groups - that was OK with old implementation where controls were checked by group, but now we rely on control names being unique in the form. This will be safe since the form transforms it to a dictionary anyway when it emits option values.